rtos/mbed_rtos_types.h

```diff
@@ -58,6 +58,62 @@ typedef enum {
     osStatusReserved        = 0x7FFFFFFF  ///< Prevents enum down-size compiler optimization.
 } osStatus_t;
 
+/// Priority values.
+typedef enum {
```

Comment:
Can't we use the values defined in CMSIS RTOS APIs?

Reply:
Can we for baremetal builds?